packages/jsii/lib/assembler.ts

```diff
@@ -3240,18 +3238,8 @@ function isSingleValuedEnum(
   return false;
 }
 
-async function findPackageInfo(
-  fromDir: string,
-): Promise<PackageJson | undefined> {
-  const filePath = path.join(fromDir, 'package.json');
-  if (await fs.pathExists(filePath)) {
-    return fs.readJson(filePath);
-  }
-  const parent = path.dirname(fromDir);
-  if (parent === fromDir) {
-    return undefined;
-  }
-  return findPackageInfo(parent);
+function findPackageInfo(fromDir: string): PackageJson | undefined {
+  return getPackageJson(fromDir);
 }
```

Comment:
That's effectively the same as:

```suggestion
const findPackageInfo = getPackageJson;
```

You should probably just get rid of that and refactor usage to `getPackageJson`.